pages/page_builder.rb

```diff
@@ -31,6 +33,12 @@ def login_page
         LoginPage.new @params #@url, @driver, @wait
     end
 
+    def new_homes_page
+        # should these be hard-coded, stored somewhere, or am I missing something?
```

Comment:
Yes, these should be hardcoded in the initialize method of the "new home" page 

I had it hardcoded in the login page but you suggested we should remove it and let the redirect take care of itself

Basically, the URL for the "page_builder" should always just be the base URL, and the page object for the page you want to use knows the "path" part, in this case "/homes_owners/new"